src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -0,0 +1,238 @@
+          { emd.getEmdDescription.getTermsTableOfContents.asScala.map(bs => <dct:description xml:lang={ lang(bs) } descriptionType='TableOfContent'>{ bs.getValue }</dct:description>) }
+          { /* instructions for reuse not specified as such in EMD */ }
+          { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:creator>) }
+          { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
+          { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
+          { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+          { disciplines.map(code => <ddm:audience>{ code }</ddm:audience>) }
+          <ddm:accessRights>{ emd.getEmdRights.getAccessCategory }</ddm:accessRights>
+        </ddm:profile>
+        <ddm:dcmiMetadata>
+          { emd.getEmdIdentifier.getDcIdentifier.asScala.filter(isDdmId).map(bi => <dct:identifier xsi:type={ xsiType(bi) }>{ bi.getValue }</dct:identifier>) }
+          { emd.getEmdTitle.getTermsAlternative.asScala.map(str => <dct:alternative>{ str }</dct:alternative>) }
+          { emd.getEmdRelation.getDCRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
+          { emd.getEmdRelation.getRelationMap.asScala.map { case (key, values) => values.asScala.map(toRelationXml(key, _)) } }
+          { emd.getEmdContributor.getDcContributor.asScala.map(bs => <dc:contributor eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dc:contributor>) }
+          { emd.getEmdContributor.getEasContributor.asScala.map(author => <dcx-dai:contributorDetails>{ toXml(author)} </dcx-dai:contributorDetails>) }
+          { /* easy-desposit-api creates authors once more as rightsHolders TODO ? */ }
+          { emd.getEmdPublisher.getDcPublisher.asScala.map(bs => <dct:publisher xml:lang={ lang(bs) }>{ bs.getValue }</dct:publisher>) }
+          { emd.getEmdSource.getDcSource.asScala.map(bs => <dc:source xml:lang={ lang(bs) }>{ bs.getValue }</dc:source>) }
+          { emd.getEmdType.getDcType.asScala.map(bs => <dct:type xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:type>) }
+          { emd.getEmdFormat.getDcFormat.asScala.map(bs => <dct:format xsi:type={ xsiType(bs) }>{ bs.getValue }</dct:format>) }
+          { emd.getEmdFormat.getTermsExtent.asScala.map(notImplemented("extent format")) }
+          { emd.getEmdFormat.getTermsMedium.asScala.map(notImplemented("medium formt")) }
+          { emd.getEmdSubject.getDcSubject.asScala.map(bs => <dc:subject xml:lang={ lang(bs) } xsi:type={ xsiType(bs) }>{ bs.getValue }</dc:subject>) }
+          { emd.getEmdCoverage.getDcCoverage.asScala.map(bs => <dct:coverage eas:scheme={ bs.getScheme } eas:schemeId={ bs.getSchemeId } xml:lang={ lang(bs) }>{ bs.getValue }</dct:coverage>) }
```

Comment:
kun je uit deze getScheme/getSchemeId afleiden of het een plaats of periode is?
in dat geval zou je `xsi:type="dct:Period"` kunnen gebruiken

```suggestion
          { emd.getEmdCoverage.getDcCoverage.asScala.map(bs => <dct:coverage xml:lang={ lang(bs) }>{ bs.getValue }</dct:coverage>) }
```